### PR TITLE
feat: allow cleanup-python-code-create-pr to read the PR description from a file

### PR DESCRIPTION
**Pull request by rgraber**
Update cleanup-python-code-create-pr to look for a file containing the desired PR description before defaulting to the current message. Allows for more customization in creating PRs from this job.

### CODE REVIEW

File: testeng/resources/cleanup-python-code-create-pr.sh [hunk 1]
```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
+  then
+     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
+  else
+     local message="$(cat <<EOF
```
Comment (robrap) [inline testeng/resources/cleanup-python-code-create-pr.sh:61]: What does this line do exactly, and do we want one or the other, or potentially both?

File: testeng/resources/cleanup-python-code-create-pr.sh [hunk 2]
```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
+  then
+     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
```
Comment (robrap) [inline testeng/resources/cleanup-python-code-create-pr.sh:59]: Is there an explanation around the choice of dropping this file in `.git`?

File: testeng/resources/cleanup-python-code-create-pr.sh [hunk 3]
```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
```
Comment (robrap) [inline testeng/resources/cleanup-python-code-create-pr.sh:57]: Could you add a version of the PR description comment in the code? I had no idea what this was doing or used for without it.

Comment (rgraber) [reply testeng/resources/cleanup-python-code-create-pr.sh:57]: I don't understand. What should the comment be about?

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 1 above)
Comment (rgraber) [reply testeng/resources/cleanup-python-code-create-pr.sh:61]: I think for now we want one or the other. There isn't really a use case for making this too customizable yet, and doing so would require non trivial changes elsewhere (to allow for more options on the jenkins job)

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 3 above)
Comment (rgraber) [reply testeng/resources/cleanup-python-code-create-pr.sh:57]: Ah I think I get it now. Was having trouble with sentence parsing

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 2 above)
Comment (rgraber) [reply testeng/resources/cleanup-python-code-create-pr.sh:59]: Seemed like a place unlikely to conflict with anything else

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 3 above)
Comment (robrap) [reply testeng/resources/cleanup-python-code-create-pr.sh:60]: Thanks!

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 1 above)
Comment (robrap) [reply testeng/resources/cleanup-python-code-create-pr.sh:64]: I still don’t know what this does.

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 2 above)
Comment (robrap) [reply testeng/resources/cleanup-python-code-create-pr.sh:62]: Ok. Doesn’t feel right to me, but I don’t have a proposal, so I guess we’re going with it, unless you ask for other ideas. 🙂

**Review by robrap — APPROVED** [review]
Still curious about one part, but doesn’t need to block this. Thanks.

File: testeng/resources/cleanup-python-code-create-pr.sh (hunk 2 above)
Comment (timmc-edx) [reply testeng/resources/cleanup-python-code-create-pr.sh:62]: It's also a place that won't conflict with the rest of the working directory.

**Pull request merged**